packages/fxa-settings/src/pages/CannotCreateAccount/index.tsx

```diff
@@ -8,19 +8,19 @@ import { RouteComponentProps } from '@reach/router';
 import AppLayout from '../../components/AppLayout';
 import CardHeader from '../../components/CardHeader';
 import { FtlMsg } from 'fxa-react/lib/utils';
-import { logPageViewEvent } from '../../lib/metrics';
+import { usePageViewEvent } from '../../lib/metrics';
+import { REACT_ENTRYPOINT } from '../../constants';
 
-export const routeName = 'cannot_create_account';
+export const viewName = 'cannot_create_account';
```

Comment:
Q: Should this use hyphens instead of underscores?
I think this was the only 1/24 of my quick Ctrl+F that was underscored for `viewName`.

```diff
-export const routeName = 'cookies_disabled';
+export const viewName = 'cookies-disabled';
```

Reply:
Sorry, I probably should have opened this as a draft since I didn't have time to review this before hopping into a meeting. I'll clean it up and push now, this should be `cookies-disabled`.